docs/deployment/getting-started-locally/getting-started-locally.md

```diff
@@ -0,0 +1,196 @@
+> ```
+
+## 02-Setting up etcd-druid
+
+### Configuring etcd-druid
+
+Prior to deploying `etcd-druid`, it can be configured via CLI-args and environment variables. 
+
+* To configure CLI args you can modify [`charts/druid/values.yaml`](https://github.com/gardener/etcd-druid/blob/3383e0219a6c21c6ef1d5610db964cc3524807c8/charts/druid/values.yaml).  For e.g. if you wish to `auto-reconcile` any change done to `Etcd` CR then you should set `enableEtcdSpecAutoReconcile` to true. By default this will be switched off.
+* `DRUID_E2E_TEST=true` : sets specific configuration for etcd-druid for optimal e2e test runs, like a lower sync period for the etcd controller.
+
+### Deploying etcd-druid
+
+Any variant of `make deploy-*` command uses [helm](https://helm.sh/) and [skaffold](https://skaffold.dev/) to build and deploy `etcd-druid` to the target Kubernetes cluster. In addition to deploying `etcd-druid` it will also install the [Etcd CRD](https://github.com/gardener/etcd-druid/blob/3383e0219a6c21c6ef1d5610db964cc3524807c8/config/crd/bases/crd-druid.gardener.cloud_etcds.yaml) and [EtcdCopyBackup CRD](https://github.com/gardener/etcd-druid/blob/3383e0219a6c21c6ef1d5610db964cc3524807c8/config/crd/bases/crd-druid.gardener.cloud_etcdcopybackupstasks.yaml).
+
+#### Regular mode
+
+```bash
+> make deploy
+```
+
+The above command will use [skaffold](https://skaffold.dev/) to build and deploy `etcd-druid`  to the k8s kind cluster pointed to by `KUBECONFIG` environment variable.
+
+#### Dev mode
+
+```bash
+> make deploy-dev
+```
+
+This is similar to `make deploy` but additionally starts a [skaffold dev loop](https://skaffold.dev/docs/workflows/dev/). After the initial deployment, skaffold starts watching source files. Once it has detected changes, you can press any key to update the `etcd-druid` deployment.
+
+#### Debug mode
+
+```bash
+> make deploy-debug
+```
+
+This is similar to `make deploy-dev` but additionally configures containers in pods for debugging as required for each container's runtime technology. The associated debugging ports are exposed and labelled so that they can be port-forwarded to the local machine. Skaffold disables automatic image rebuilding and syncing when using the `debug` mode as compared to `dev` mode.
+
+Go debugging uses [Delve](https://github.com/go-delve/delve). Please see the [skaffold debugging documentation](https://skaffold.dev/docs/workflows/debug/) how to setup your IDE accordingly. 
+
+> **Note:** Resuming or stopping only a single goroutine (Go Issue [25578](https://github.com/golang/go/issues/25578), [31132](https://github.com/golang/go/issues/31132)) is currently not supported, so the action will cause all the goroutines to get activated or paused.
+
+This means that when a goroutine is paused on a breakpoint, then all the other goroutines are also paused. This should be kept in mind when using `skaffold debug`.
+
+## 03-Configure Backup [*Optional*]
+
+> **Note:** If you wish to do not backup any snapshots
```

Comment:
This note seems incomplete. Did you want it to tell readers to not set `Etcd.Spec.Backup.Store` in order to disable snapshotting?

Reply:
removed it. I think this was a leftover